<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZarzadzanieKlientami</class>
 <widget class="QWidget" name="ZarzadzanieKlientami">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTableView" name="tablica"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="Id_Label">
           <property name="text">
            <string>Id Klienta</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Id_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="Imie_Label">
           <property name="text">
            <string>Imię</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Imie_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="Nazwisko_Label">
           <property name="text">
            <string>Nazwisko</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Nazwisko_Line"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="Telefon_Label">
           <property name="text">
            <string>Telefon</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Telefon_Line"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Wroc_Button">
         <property name="text">
          <string>Wróć</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Dodaj_Button">
         <property name="text">
          <string>Dodaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Edytuj_Button">
         <property name="text">
          <string>Edytuj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Usun_Button">
         <property name="text">
          <string>Usuń</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
